Programowanie Obiektowe/ProgramowanieObiektoweReszta/_1_11Struktury/App1/App1.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace _1_11Struktury.App1
+{
+    internal static class App1
+    {
+        public static void Start()
+        {
+            var todos = new List<Todo>();
+            while (true)
```

Comment:
To się nigdy nie skończy, chyba że wyrzucisz wyjątek

Reply:
Zastosowałem takie rozwiązanie:
`            switch (taskNumber)
            {
                case '1':
                    return AddTodo(todos);
                case '2':
                    DisplayTodos(todos);
                    break;
                case '3':
                    return DeleteTodo(todos);
                default:
                    Environment.Exit(0);
                    break;
            }`

Reply:
Przerobiłem używając słowa kluczowego `out`
[58be68a](https://github.com/broniuu/Praktyki2022/pull/2/commits/58be68aea89536f6e72268131e5f050fc3c6964e)